### PR TITLE
Clarify ACL terminology

### PR DESCRIPTION
**Pull request by Vinnl**
Virginia pointed out that the current ACL explanation isn't very helpful, and suggested that an example might help. Down the road we might want to show actual Turtle, but this hopefully makes things a bit clearer on the short term.

Can be seen here: https://docs-client-lib-js-git-docs-glossary-acl-extension.inrupt.vercel.app/reference/glossary/#term-ACL

### CODE REVIEW

File: source/reference/glossary.rst [hunk 1]
```diff
@@ -9,8 +9,15 @@ refer to `<https://solidproject.org/faqs>`_.
 
 
    ACL
-      An **Access Control List**. ACL entry specifies the access
-      granted to an agent to a resource.  See also :term:`Access Modes`.
+      An **Access Control List**, a :term:`Resource` with data about
+      another Resource. Specifically, it controls who has what level of
```
Comment (kay-kim) [inline source/reference/glossary.rst:13]: maybe combine the first two sentences to a little more specific for "with data about another Resource" ?

An **Access Control List** is a :term:`Resource` that controls users' level of access to a Resource. For example, ...

Comment (kay-kim) [reply source/reference/glossary.rst:13]: I wonder if we should try to squeeze in 

`{ read: <boolean>, append: <boolean>, write: <boolean>, control: <boolean> }`

Like 

An Access Control List is a Resource that controls users's level of of access (`{ read: <boolean>, append: <boolean>, write: <boolean>, control: <boolean> }`) to a Resource

Comment (Vinnl) [reply source/reference/glossary.rst]: I added it in right now: https://docs-client-lib-js-dov34mn63.vercel.app/reference/glossary/

What do you think? Too cramped?

Comment (kay-kim) [reply source/reference/glossary.rst]: It is a little bit - but eh, I think it's okay for now.  I'll create a DOCS JIRA ticket to vet the Glossary - and schedule the work for this week.

**Review by kay-kim — APPROVED** [review]
LGTM --
created https://inrupt.atlassian.net/browse/DOCS-174  to flesh out the glossary page a bit.

**Pull request merged**